internal/installation/stop.go

```diff
@@ -35,5 +63,66 @@ func StopRunning(installPath string) (rerr error) {
 		}
 	}
 
+	procs, err := process.Processes()
+	if err != nil {
+		return errs.Wrap(err, "Could not get list of running processes")
+	}
+
+	// This is a bit heavy handed but ensures that there are
+	// no running state-svc processes which could lead to
+	// errors when updating
+	for _, p := range procs {
+		n, err := p.Name()
+		if err != nil {
+			logging.Error("Could not get process name: %v", err)
+			continue
+		}
+
+		if n == constants.ServiceCommandName {
+			exe, err := p.Exe()
+			if err != nil {
+				logging.Error("Could not get executable path for state-svc process, error: %v", err)
+				continue
+			}
+
+			if !strings.Contains(strings.ToLower(exe), "activestate") {
+				logging.Error("Found state-svc process in unexpected directory: %s", exe)
+				continue
+			}
+
+			logging.Debug("Found running state-svc process: %d", p.Pid)
+			err = stopSvcProcess(p)
+			if err != nil {
+				return errs.Wrap(err, "Could not stop service process")
+			}
+		}
+	}
+
 	return nil
 }
+
+func stopSvcProcess(proc *process.Process) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+	defer cancel()
+
+	signalErrs := make(chan error)
+	go func() {
+		signalErrs <- proc.SendSignal(syscall.SIGTERM)
+	}()
+
+	select {
+	case err := <-signalErrs:
+		if err != nil {
+			return errs.Wrap(err, "Could not send SIGTERM to service process")
+		}
+		logging.Debug("Stopped state-svc process with SIGTERM")
+		return nil
+	case <-ctx.Done():
```

Comment:
nit: If this is just acting as a timeout within this function and the context type isn't really needed to be passed around, I suggest:
```suggestion
	signalErrs := make(chan error)
	go func() {
		signalErrs <- proc.SendSignal(syscall.SIGTERM)
	}()

	select {
	case err := <-signalErrs:
		if err != nil {
			return errs.Wrap(err, "Could not send SIGTERM to service process")
		}
		logging.Debug("Stopped state-svc process with SIGTERM")
		return nil
	case <-time.After(time.Second):
```